package-lock.json

```diff
@@ -4563,6 +4583,12 @@
           "requires": {
             "ms": "^2.1.1"
           }
+        },
+        "semver": {
+          "version": "6.3.0",
```

Comment:
It looks like `superagent` is also using semver@6, you will also need to upgrade `superagent` version requirement to `^5.3.1`